Implementacija/savetovaliste/app/Models/KorisnikOcenioOdgovorModel.php

```diff
@@ -0,0 +1,25 @@
+<?php namespace App\Models;
+
+use CodeIgniter\Model;
+use App\Models\PitanjeModel;
+
+class KorisnikOcenioOdgovorModel extends Model
+{
+        protected $table      = 'korisnik_ocenio_odgovor';
+        protected $allowedFields = ['ocena'];
+        protected $returnType = 'object';
+        
```

Comment:
$useAutoIncrement = false, da ne bi id-jeve slučajno inkrementirao kad ne treba.

```suggestion
    protected $useAutoIncrement = false;
```

Reply:
Nisam sigurna da li je neophodno, ali sam promenila na to

Reply:
E, moguće da nije, al neka ga za svaki slučaj...